packages/agent-toolkit/src/core/tools/platform-api-tools/index.ts

```diff
@@ -36,20 +37,4 @@ export const allTools = [
   FetchCustomActivityTool,
 ];
 
-export * from './delete-item-tool';
-export * from './get-board-items-tool';
-export * from './create-item-tool';
-export * from './create-update-tool';
-export * from './get-board-schema-tool';
-export * from './get-users-tool';
-export * from './change-item-column-values-tool';
-export * from './move-item-to-group-tool';
-export * from './create-board-tool';
-export * from './create-column-tool';
-export * from './delete-column-tool';
-export * from './all-monday-api-tool';
-export * from './get-graphql-schema-tool';
-export * from './get-type-details-tool';
-export * from './create-custom-activity-tool';
-export * from './create-timeline-item-tool';
-export * from './fetch-custom-activity-tool';
+export { BaseMondayApiTool };
```

Comment:
why did you delete all tools? I think we should export them as well, also removing an import is a breaking change

Reply:
also why did you exported the base?

Reply:
Ill specifically import